Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/response/StandardInboundPatientDiscoveryDeferredResponseTest.java

```diff
@@ -280,7 +280,7 @@ public void auditOffForInboundPDDeferredResp() {
 
         // Verify audits
         verify(mockEJBLogger, never()).auditResponseMessage(eq(request), eq(actualResponse), eq(assertion),
-            isNull(NhinTargetSystemType.class), eq(NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION),
+            isNull(NhinTargetSystemType.class), eq(NhincConstants.AUDIT_LOG_INBOUND_DIRECTION),
             eq(NhincConstants.AUDIT_LOG_NHIN_INTERFACE), eq(Boolean.FALSE), eq(webContextProperties),
             eq(NhincConstants.PATIENT_DISCOVERY_DEFERRED_RESP_SERVICE_NAME),
             any(PatientDiscoveryDeferredResponseAuditTransforms.class));
```

Comment:
The interface is not used anymore. Can you create a ticket in Backlog to remove the parameter interface passed across all services in audit call?